tools/run_danesfield.py

```diff
@@ -474,8 +505,8 @@ def main(args):
     cmd_args.append('--input_obj_paths')
     obj_list = glob.glob("{}/*.obj".format(roof_geon_extraction_outdir))
     # remove occlusion_mesh and results (building_<i>.obj)
```

Comment:
remove the building filter but keep the occlusion_mesh filter.  Then update the comment above to say `remove occlusion_mesh`